### ISSUE TITLE
Getting this error a bunch now: Error in stri_locate_first_regex(string, pattern, opts_regex = attr(pattern,  :  Syntax error in regexp pattern. (U_REGEX_RULE_SYNTAX)

### ISSUE THREAD

**Issue by abresler** [tidyverse/rvest#32]
Was testing this toy example but have gotten it a bunch lately when using html_nodes.  Seems like this maybe coming from a dependancy but hoping someone knew of a fix.  Here is the toy example not working using the newest rvest:

library(rvest)    # devtools::install_github("rvest","hadley")
library(stringr)  # install.pacakges("stringr")

url <- "http://colleges.usnews.rankingsandreviews.com/best-colleges/rankings/national-universities/data/page+%d"

trimNewline <- function(p) str_replace(p, "\n","")
asInteger <- function(x) as.integer(str_replace(x,",",""))
fromPercent <- function(x) as.numeric(str_replace(x, "%", ""))/100

table <- 1:3 %>%
    lapply(function(page) {
        nodes <- sprintf(url, page) %>%
            html() %>%
            html_nodes("table tbody tr")

```
    column <- function(xpath) nodes %>% html_node(xpath = xpath) %>% html_text(trim = TRUE)

    data.frame(
        rank = column("td[1]/div[1]/span") %>%
            str_replace("#(\\d+)(Tie)?","\\1") %>%
            as.integer,
        score = column("td[1]/span[1]/span") %>%
            str_replace("(\\d+) out of 100.","\\1") %>%
            asInteger,
        name = column("td[2]/a"),
        location = column("td[2]/p/text()[1]"),
        tuitionAndFees = column("td[3]/text()[1]") %>% trimNewline,
        totalEnrollment = column("td[4]/text()[1]") %>% asInteger,
        fall2013AcceptanceRate = column("td[5]/text()[1]") %>% fromPercent,
        averageFreshmanRetentionRate = column("td[6]/text()[1]") %>% fromPercent(),
        sixYearGraduationRate = column("td[7]/text()[1]") %>% fromPercent,
        stringsAsFactors = FALSE
    )
}) %>%
do.call(rbind, .)
```

**Comment by cawthm** [comment]
I am having this same error message mentioned above, both in my specific use case, but also in the Zillow example above.

I've updated both rvest (v0.2.0, from the development repo) and stringr (v1.0.0, from CRAN). 

Any advice would be appreciated.

**Comment by dbuijs** [comment]
I said my issue was resolved by upgrading to R 3.2.0, but I actually rebuilt the `rocker/ropensci`  Docker image, that updated everything. Before I did a complete rebuild, I tried updating first `rvest` and then `stringr`, but that didn't solve the issue.
